includes/cli/class-flush-command.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-flush-command.php, but found phpcs-scan-3yuYoP.php (*WordPress.Files.FileName.InvalidClassFileName*).